doc/developer-guides/coding_guidelines.rst

```diff
@@ -1431,7 +1366,7 @@ Compliant example::
        }
 
 
-ST-18:  ++ or -- operation shall be used with restrictions
+ST-15:  ++ or -- operation shall be used with restrictions
```

Comment:
When we have tools that enforce these guidelines and report the rule number violation, we can no longer change the numbers.